tests/foreman/cli/test_import.py

```diff
@@ -0,0 +1,66 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Host Collection CLI"""
+
+from ddt import ddt
+from robottelo.test import CLITestCase
+from robottelo.cli.import_ import Import
+
+
+@ddt
+class TestImport(CLITestCase):
+    """Import CLI tests."""
+
+    """All default tests pass no options to the imprt object
+    In such case methods download a default data set from URL
+    specified in robottelo.properties.
+
+    """
+
+    def test_import_orgs_default(self):
+        """@test: Import all 3 organizations from our default data set
+        (predefined source).
+
+        @feature: Import Organizations
+
+        @assert: 3 Organizations are created
+
+        """
+
+        rc = Import.organization()
+        exp_output = [u'Summary', u'  Created 3 organizations.', u'']
+        self.assertEqual(rc.return_code, 0)
+        self.assertEqual(rc.stdout, exp_output)
+
+    def test_import_users_default(self):
+        """@test: Import all 3 users from the our default data set (predefined
+        source).
+
+        @feature: Import Users
+
+        @assert: 3 Users created
+
+        """
+
+        rc = Import.user()
+        exp_output = [u'Summary', u'  Created 3 users.', u'']
+        self.assertEqual(rc.return_code, 0)
+        self.assertEqual(rc.stdout, exp_output)
+
+    def test_reimport_orgs_default(self):
+        """@test: Try to Import all organizations from the
+        predefined source and try to import them again
+
+        @feature: Import Organizations twice
+
+        @assert: 2nd Import will result in No Action Taken
+
+        """
+
+        # ensure the organizations will exist before 2nd call
+        rc1 = Import.organization()
+        self.assertEqual(rc1.return_code, 0)
+
+        # try to import already existent orgs
+        rc2 = Import.organization()
+        exp_output = [u'Summary', u'  No action taken.', u'']
+        self.assertEqual(rc2.stdout, exp_output)
```

Comment:
Please avoid single-use variables when reasonably possible. A better way to write this logic is as follows:

```python
self.assertEqual(Import.organization().return_code, 0)
self.assertEqual(
    Import.organization().stdout,
    [u'Summary', u'  No action taken.', u'']
)
```